tests/src/test/scala/akka/kafka/scaladsl/CommittingSpec.scala

```diff
@@ -51,7 +51,7 @@ class CommittingSpec extends SpecBase with TestcontainersKafkaLike with Inside {
       val (control, probe1) = Consumer
         .committableSource(consumerSettings, Subscriptions.topics(topic1))
         .mapAsync(10) { elem =>
-          elem.committableOffset.commitInternal().map { _ =>
+          elem.committableOffset.commitInternal(flush = false).map { _ =>
```

Comment:
With the default param already `false` you can drop most of these existing test updates.

Reply:
Removed redundant default arguments (also inother places)